app/views/courses/apprenticeship/_form_fields.html.erb

```diff
@@ -1,5 +1,5 @@
 <div
-  class="<%= cns("govuk-form-group", "govuk-form-group--error": @errors && @errors[:funding_type]&.any?) %>"
+  class="<%= cns("govuk-form-group", "govuk-form-group--error": @errors&.[](:funding_type)&.any? || @errors&.[](:program_type)&.any?) %>"
```

Comment:
This doesn't feel great to read and would potentially be quite confusing (A mixture of both multiple `&.`s and the `[](:key)` makes it quite hard to understand what's happening. Wonder if there's a helper method we could extract for this common pattern :thinking: